cs-fun-practice.rb

```diff
@@ -0,0 +1,50 @@
+# PROBLEM 2
+
+def stretch(array)
+  new_array = []
+  array.length.times do |a|
+    if array[a] % 2 == 0
+      2.times do
+        new_array << array[a] / 2
+      end
+    else
+      new_array << ((array[a] / 2) + 1)
+      new_array << array[a]/2
+    end
+  end
+  return new_array
+end
```

Comment:
This looks good. 

nit-picky: I think using |i| instead of |a| is more common, though it doesn't really matter. i is typically used for lots of reasons, but here are some: http://stackoverflow.com/questions/4137785/why-are-variables-i-and-j-used-for-counters